packages/system/src/Name/Components/CustomerSearchModal/CustomerSearchModal.tsx

```diff
@@ -18,7 +23,13 @@ export const CustomerSearchModal: FC<NameSearchProps> = ({
       options={data?.nodes ?? []}
       onClose={onClose}
       title={t('customers')}
-      optionKey="name"
+      renderOption={NameOptionRenderer}
+      getOptionLabel={(option: NameRowFragment) => option.name}
```

Comment:
adding the option label, as Autocomplete was unhappy with `[object, object]` as the label 🙄